chr.clj

```diff
@@ -23,8 +23,12 @@
   (subs line 15))
 
 (defn- parse-commands [line]
-  (let [rest (second (string/split line #";"))]
```

Comment:
avoid naming your binding `rest` as it can cause confusion with `clojure.core/rest`